service/history/execution/context.go

```diff
@@ -45,6 +45,7 @@ import (
 
 const (
 	defaultRemoteCallTimeout = 30 * time.Second
+	buffertimeforTTL         = 15
```

Comment:
Would you mind changing this to be a value which indicates the value it's on, I assume it's days? 

`ttlBufferDays` might be one option?